x-pack/plugins/alerting/server/task_runner/rule_action_helper.test.ts

```diff
@@ -153,6 +162,17 @@ describe('rule_action_helper', () => {
       expect(result).toEqual({ '111-111': { date: new Date('01.01.2020').toISOString() } });
     });
 
+    test('should filtered out system actions', () => {
```

Comment:
Should we add `mockSystemAction`?

Reply:
resolve since this function is not use for system action since we separated the attributes of actions and systemActions